examples/provider/bid/conversant/conversant-ex1.html

```diff
@@ -0,0 +1,379 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Conversant - PubFood Example page</title>
+  </head>
+  <body>
+    <script src="../../../../build/pubfood.js"></script>
+    <script  type="text/javascript">
+
+      //add the following lines to 'Code Snippet' box of DFP third party creative within a <script> tag
+      //gpt would then call renderAd below to render the ad
+
+     /*
+      var w = window;
+      for (i = 0; i < 10; i++) {
+        w = w.parent;
+        try {
+          w.renderAd(document, '%%PATTERN:cn728x90adId%%');
+          break;
+        } catch(e) {continue;}
+      }
+     */
+
+      //rudimentary html validation, use stricter validation if needed
+      var validateHTML = function (snippet) {
+        return (snippet && snippet.startsWith('<') && snippet.endsWith('>'));
+      };
+      //use similar logic to render ad. dfp would call with document and id
+      window.renderAd = function (doc, id) {
```

Comment:
`window.renderAd()` needs to be namespaced here. A BidProvider delegate specific namespace e.g. something like `window.cnRenderAd()` or whatever would be fine. However, for a more general namespace, it would be acceptable to add the render function to the `pubfood` object for more general use.

i.e. `window.pubfood.renderAd()`

Great that you have a comment in the example code to illustrate an example LineItem creative tag. Let's get this `renderAd` function a namespace.

The industry has evolved from the infancy of Pubfood API and response from the project wrt rendering is required. To that end, have added #77 to capture this and set an initiative in motion.